.changeset/four-terms-reflect.md

```diff
@@ -0,0 +1,5 @@
+---
+"live-mobile": patch
+---
+
+Android: bump targetSdkVersion and compileSdkVersion to 31
```

Comment:
Just curious why two changesets

Reply:
idk I just used "pnpm changelog" instead of editing the existing one 🤔
Is it bad to have 2?

Reply:
I asked DX and it's actually a good practice